src/sx/package.json

```diff
@@ -14,11 +14,13 @@
     "@adeira/murmur-hash": "^1.0.0",
     "@adeira/signed-source": "^1.0.0",
     "@babel/runtime": "^7.12.5",
+    "autoprefixer": "^10.0.4",
```

Comment:
Will this be a part of the runtime-bundle? because it is quite huge

https://bundlephobia.com/result?p=autoprefixer@10.0.4

Reply:
It's used in https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss do you see any issue there?

Reply:
This is production build of relay-example: 

<img width="700" alt="Screenshot 2020-12-06 at 14 52 35" src="https://user-images.githubusercontent.com/7705041/101282005-c0aaa200-37d2-11eb-889f-61202d1c15c4.png">

Reply:
And this is current master:

<img width="693" alt="Screenshot 2020-12-06 at 14 54 54" src="https://user-images.githubusercontent.com/7705041/101282061-0a938800-37d3-11eb-89b1-57a042956f36.png">

Reply:
It is true that during a build of `sx-tailwind-website` in the monorepo I get warnings and increased bundle size:
```
/Users/michalsanger/projects/adeira/universe/node_modules/@adeira/sx/node_modules/browserslist/node.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

/Users/michalsanger/projects/adeira/universe/node_modules/@adeira/sx/node_modules/browserslist/node.js 175:18-177:7
Critical dependency: the request of a dependency is an expression
```

But when I tried `postcss` and `autoprefixer` on a standalone Next.js project everything was fine. Maybe because of `next.config.js`?

Reply:
I guess it depends on how it is used. In our case, I am pretty sure using it in the stylecollector will make it a part of the runtime bundle, and I would prefer some duplicated CSS properties in my stylesheet over a +1MB addition to the runtime bundle 😬

Reply:
So is there some cheaper alternative for CSS-in-JS? 1 MB is really a lot. Why is it so big? 🤔

Reply:
Maybe the PostCSS is designed to be used with Babel & Webpack, not during a runtime. I'll try to use https://github.com/robinweser/inline-style-prefixer in a new PR but probably the output will not be that good.

Reply:
I think most `CSS-in-JS` uses [styleis](https://www.npmjs.com/package/stylis)

Reply:
Alternatively, you could read it from `mdn/browser-compat-data` as suggested here: https://github.com/adeira/universe/issues/996

Reply:
Attempt with Stylis #1480